include/mme-app/mmeStatsPromClient.h

```diff
@@ -28,6 +28,8 @@ enum class mmeStatsCounter {
 	MME_MSG_RX_NAS_AUTHENTICATION_RESPONSE,
 	MME_MSG_RX_NAS_SECURITY_MODE_RESPONSE,
 	MME_MSG_RX_NAS_ESM_RESPONSE,
+	MME_MSG_RX_NAS_ACT_DED_BR_CTXT_ACPT,
```

Comment:
This is autogenerated code. I need to check if you have updated the original json file.

Reply:
Updated the JSON File